Documentation/botr/readytorun-format.md

```diff
@@ -4,10 +4,11 @@ ReadyToRun File Format
 Revisions:
 * 1.1 - [Jan Kotas](https://github.com/jkotas) - 2015
 * 3.1 - [Tomas Rylek](https://github.com/trylek) - 2019
+* 3.2 - [Tomas Rylek](https://github.com/trylek) - 2019 - *I'm not clear on whether we need to bump this up to 4.0*
```

Comment:
I don't believe we'll need to bump to 4.0 unless a 3.0 format file is no longer loadable with our R2R loader.

Reply:
I think I was rather concerned by the fact that, as the 3.1 loader doesn't know about the new proposed COMPOSITE flag, it wouldn't recognize the composite file for what it is and load it incorrectly. I think this has mostly become moot with @jkotas` suggestion to drop the global COR header in the composite case as in such case a legacy loader has no way of trying to make sense of the new file.